docker/hsm.Dockerfile

```diff
@@ -1,6 +1,16 @@
 # Use the amazon linux image
 FROM amazonlinux:2
 
+# Install SoftHSM2
+RUN yum install -y softhsm
```

Comment:
Would this start any background "softhsm" service? I'm not familiar with this package, just asking. If softhsm2 will run a softhsm process I think we should maybe consider running it in separated containers, and make ethsigner connect to it over the network (if this is possible and makes sense for how softhsm works)